tests/test_frame.py

```diff
@@ -42,6 +42,7 @@ def test_types_init() -> None:
         dtype=np.int8,
         copy=True,
     )
+    pd.DataFrame(0, index=[0, 1], columns=[0, 1])
```

Comment:
can you do `check(assert_type(pd.DataFrame(0, index=[0, 1], columns=[0, 1]), pd.DataFrame), pd.DataFrame)` here?  

I know we don't have it above, but we're putting this in with all new tests.